addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -94,6 +94,34 @@ export class WebsitePreview extends Component {
             this.websiteService.pageDocument = null;
         });
 
+        /**
+         * This removes the 'Odoo' prefix of the title service to display
+         * cleanly the frontend's document title (see _replaceBrowserUrl), and
+         * replaces the backend favicon with the frontend's one.
+         * These changes are reverted when the component is unmounted.
+         */
+        useEffect(() => {
+            const backendIconEl = document.querySelector("link[rel~='icon']");
+            // Save initial backend values.
+            const { href: backendIconHref } = backendIconEl;
```

Comment:
With that kind of logic, you can obfuscate all lines of the code. If later we need more values from `backendIconEl`... we'll change the code to the most readable version which allows to extract values. Meanwhile I don't see the point in using an unreadable version to read one attribute :/ You really don't agree that `const backendIconHref = backendIconEl.href` is more readable?

Reply:
No, I don't agree, I think it's appropriate to use the es6 destructuring assignment here, but I'll follow your call :+1:

Reply:
Ok, then I'll guess I'll try not to make that kind of feedback anymore. But I really really don't understand :grimacing: (but I don't like that form even with multiple values TBH, I feel it should be the other way around (`backendIconHref: href`) so it can be read `myVariable = theValue` = "create variables whose value is this attribute from this object"). But there are 46 occurences exactly like this in the codebase, so...

I guess many people can read

"create a variable" "with the href property" "to be named backendIconHref" "to be set" "using the backendIconEl to read the property"

The same way as

"create a variable" "named backendIconHref" "to be set" "from the backendIconEl variable" "reading the href property"

Reply:
I guess it's a matter of habit, I got used to use destructuring reading it as you described it, working with react, most of the time component will be defined as:
```js
const Greetings = ({ name, lastName }) => {
  return <p>Hello {name} {lastName}<p>
}
```
instead of 
```js
const Greetings = (props) => {
  return <p>Hello {props.name} {props.lastName}<p>
}
```

Reply:
It's not the same form here. I don't mind destructuring with `const {name} = props` because it that case, it reads exactly as it is (even more true with function params ("take these properties from the object given as argument")). It's really the form `const {name: myVariable} = props` I have issues with.